isis/src/base/objs/Spice/Spice.cpp

```diff
@@ -171,14 +160,23 @@ namespace Isis {
     }
     else {
       *m_endTimePadding = 0.0;
-    }
+    }  
 
-    m_usingNaif = !lab.hasObject("NaifKeywords") || noTables;
+    // We should remove this completely in the near future 
+    // m_usingNaif = !lab.hasObject("NaifKeywords") || noTables;
+    m_usingNaif = true; 
 
+    json isd; 
     //  Modified  to load planetary ephemeris SPKs before s/c SPKs since some
     //  missions (e.g., MESSENGER) may augment the s/c SPK with new planet
     //  ephemerides. (2008-02-27 (KJB))
     if (m_usingNaif) {
+      std::string aleIsdStr = ale::load(cube.fileName().toStdString(), "", "isis");
```

Comment:
This will call ALE every time the cube is instantiated. We only want to call ALE in spiceinit.

Reply:
Defaulted `m_usingNaif` back to normal implementation, for now, should only create ISD on spiceinit.